bin/create-docs-pr.js

```diff
@@ -281,7 +284,7 @@ async function createPR(username, version, branch, dryRun, repoOwner) {
   const github = new Github(repoOwner, 'docs-website')
   const title = `Node.js Agent ${version} Release Notes`
   const prOptions = {
-    head: `${username}:${branch}`,
+    head: `${branch}`,
```

Comment:
this won't work when we run locally but it seems like repoOwner and username were redundant in this case right?

Reply:
I've updated this to run locally, though it's using repoOwner for the fork instead of username.